src/components/POLine/Location/validate.js

```diff
@@ -0,0 +1,104 @@
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+import { get } from 'lodash';
+import {
+  ERESOURCES,
+  INVENTORY_RECORDS_TYPE,
+  OTHER,
+  PHRESOURCES,
+} from '../const';
+
+export const REQUIRED_LOCATION_QUANTITY_MAP_TO_INVENTORY = {
+  [INVENTORY_RECORDS_TYPE.all]: true,
+  [INVENTORY_RECORDS_TYPE.instance]: false,
+  [INVENTORY_RECORDS_TYPE.instanceAndHolding]: true,
+  [INVENTORY_RECORDS_TYPE.none]: false,
+};
+
+export const isLocationPhysicalQuantityRequired = (orderFormat, inventory) => {
+  return [...PHRESOURCES, OTHER].includes(orderFormat)
+    ? REQUIRED_LOCATION_QUANTITY_MAP_TO_INVENTORY[inventory]
```

Comment:
whe should follow one pattern: operator on next line or on previous, I see we use both now

Reply:
I think this one is correct